src/conduit/openapi.py

```diff
@@ -13,3 +16,13 @@ def includeme(config: Configurator) -> None:
         os.path.join(os.path.dirname(__file__), "openapi.yaml"), route="/openapi.yaml"
     )
     config.pyramid_openapi3_add_explorer(route="/api")
+
+
+@exception_view_config(Exception)
+def error_response(exc: Exception, request: Request):
+    """Catch any uncaught errors and respond with a nice JSON error.
+
+    Without this, Exceptions that get to WSGI level return HTML that says
```

Comment:
```suggestion
    Without this, Exceptions that gets to WSGI level return HTML that says
```
I'm not 100% but I would put `gets` here (because exceptions is third person).

Reply:
Exception gets, Exceptions get. It's plural.